packages/@glimmer/manager/lib/internal/defaults.ts

```diff
@@ -0,0 +1,37 @@
+import { buildCapabilities } from '../util/capabilities';
+
+import type { CapturedArguments as Arguments, HelperCapabilities } from '@glimmer/interfaces';
+
+type FnArgs<Args extends Arguments = Arguments> =
+  | [...Args['positional'], Args['named']]
+  | [...Args['positional'], {}];
+
+interface FunctionHelperState<Args extends Arguments = Arguments> {
+  fn: <Return>(...args: FnArgs<Args>) => Return;
+  args: Args;
+}
+
+export class FunctionHelperManager<State extends FunctionHelperState> {
+  capabilities = buildCapabilities({
+    hasValue: true,
+    hasDestroyable: false,
+    hasScheduledEffect: false,
+  }) as HelperCapabilities;
+
+  createHelper(fn: State['fn'], args: State['args']) {
+    return { fn, args };
+  }
+
+  getValue({ fn, args }: State) {
+    let argsForFn: FnArgs<Arguments> = [
+      ...args.positional,
+      Object.keys(args.named).length > 0 ? args.named : {},
+    ];
+
+    return fn(...argsForFn);
+  }
```

Comment:
What is this used for and why do we need it? What happens if you don't have this?